drivers/scheduler/k8s/k8s.go

```diff
@@ -47,6 +47,8 @@ const (
 	// SystemdSchedServiceName is the name of the system service resposible for scheduling
 	// TODO Change this when running on openshift for the proper service name
 	SystemdSchedServiceName = "kubelet"
+	// ResizeSupported is the parameter key for the annotation when a pvc is resizeble or not
+	ResizeSupported = "torpedo/resize-supported"
```

Comment:
`ResizeSupported` doesn't need to be exported. Call it `resizeSupportedAnnotationKey`